sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -47,188 +47,194 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-ai-formrecognizer</artifactId>
-        <version>3.0.4</version>
+        <version>3.0.6</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-ai-textanalytics</artifactId>
-        <version>5.0.2</version>
+        <version>5.0.5</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core</artifactId>
-        <version>1.12.0</version>
+        <version>1.15.0</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-amqp</artifactId>
-        <version>2.0.1</version>
+        <version>2.0.4</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-core-management</artifactId>
-        <version>1.1.0</version>
+        <artifactId>azure-core-http-netty</artifactId>
+        <version>1.9.1</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-core-http-netty</artifactId>
-        <version>1.7.1</version>
+        <artifactId>azure-core-http-okhttp</artifactId>
+        <version>1.6.1</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-core-http-okhttp</artifactId>
-        <version>1.4.1</version>
+        <artifactId>azure-core-management</artifactId>
+        <version>1.2.1</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-gson</artifactId>
-        <version>1.0.4</version>
+        <version>1.1.1</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-jackson</artifactId>
-        <version>1.1.1</version>
+        <version>1.2.2</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-test</artifactId>
-        <version>1.5.2</version>
+        <version>1.6.1</version>
       </dependency>
 
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-tracing-opentelemetry</artifactId>
-        <version>1.0.0-beta.6</version>
+        <version>1.0.0-beta.9</version>
```

Comment:
We also released GA versions of `azure-communication-*` libraries. Those should be included in this bom as well.